af/oometa/006-test-lists.install.sql

```
@@ -0,0 +1,42 @@
+BEGIN;
+
+select _v.register_patch( '006-test-lists', ARRAY[ '005-repeated-report' ], NULL );
+
+CREATE SEQUENCE IF NOT EXISTS cat_no_seq;
```

Drop `IF NOT EXISTS` please. The code should rather fail than continue in unexpected condition.